M2/Macaulay2/d/actors2.dd

```diff
@@ -660,21 +660,162 @@ unSingleton(e:Expr):Expr := (
      else e);
 setupfun("unsequence",unSingleton);
 
+disassembleCode(c:Code):Expr := ( -- add locations?
```

Comment:
Since you're editing almost all of the code for `disassemble` and `pseudocode`, I would be very happy if you moved it out of `actors2.dd`, perhaps `debugging.dd` or something like that?

Reply:
what's the `.dd` stand for? (as opposed to `.d`)

Reply:
`.d` is translated to `.c` programs, `.dd` to `.cc` (cpp).